Panic using expand CLI

**Describe the bug**

A panic when using expand CLI

**Reproducing the bug**

Steps to reproduce the behavior:

```
> keto expand owner projects some_id
neither flag --read-remote nor env var KETO_READ_REMOTE are set, falling back to 127.0.0.1:4466
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0xf7ee77]

goroutine 1 [running]:
github.com/ory/keto/internal/expand.(*Tree).String(0x0, 0x13ec300, 0xc00000e018)
        /home/ory/internal/expand/tree.go:165 +0x37
github.com/ory/x/cmdx.PrintJSONAble(0xc00047a780, 0x13ea1e0, 0x0)
        /go/pkg/mod/github.com/ory/x@v0.0.212/cmdx/printing.go:119 +0x19e
github.com/ory/keto/cmd/expand.NewExpandCmd.func1(0xc00047a780, 0xc000263b90, 0x3, 0x3, 0x0, 0x0)
        /home/ory/cmd/expand/root.go:60 +0x30b
github.com/spf13/cobra.(*Command).execute(0xc00047a780, 0xc000263b60, 0x3, 0x3, 0xc00047a780, 0xc000263b60)
        /go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xc0003ef680, 0xc00003a060, 0x1405518, 0xc0007d29c0)
        /go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
github.com/spf13/cobra.(*Command).ExecuteContext(...)
        /go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:890
github.com/ory/keto/cmd.Execute()
        /home/ory/cmd/root.go:72 +0xbc
main.main()
        /home/ory/main.go:26 +0x52
```


**Expected behavior**

No panic

**Environment**

- Version: v0.6.0-alpha.1

**Additional context**

Reproducible in oasis

--- a/cmd/expand/root.go
+++ b/cmd/expand/root.go
@@ -61,7 +61,7 @@ func NewExpandCmd() *cobra.Command {
 			switch flagx.MustGetString(cmd, cmdx.FlagFormat) {
 			case string(cmdx.FormatDefault), "":
 				if tree == nil && !flagx.MustGetBool(cmd, cmdx.FlagQuiet) {
-					_, _ = fmt.Fprintln(cmd.OutOrStdout(), "Got an empty tree. This probably means that the requested relation tuple is not present in Keto.")
+					_, _ = fmt.Fprint(cmd.OutOrStdout(), "Got an empty tree. This probably means that the requested relation tuple is not present in Keto.")
 				}
 				_, _ = fmt.Fprintln(cmd.OutOrStdout())
 			}

--- a/cmd/expand/root.go
+++ b/cmd/expand/root.go
@@ -60,6 +60,9 @@ func NewExpandCmd() *cobra.Command {
 			cmdx.PrintJSONAble(cmd, tree)
 			switch flagx.MustGetString(cmd, cmdx.FlagFormat) {
 			case string(cmdx.FormatDefault), "":
+				if tree == nil && !flagx.MustGetBool(cmd, cmdx.FlagQuiet) {
+					_, _ = fmt.Fprintln(cmd.OutOrStdout(), "Got an empty tree. This probably means that the requested relation tuple is not present in Keto.")
+				}
 				_, _ = fmt.Fprintln(cmd.OutOrStdout())
 			}
 			return nil
@@ -68,6 +71,7 @@ func NewExpandCmd() *cobra.Command {
 
 	client.RegisterRemoteURLFlags(cmd.Flags())
 	cmdx.RegisterJSONFormatFlags(cmd.Flags())
+	cmdx.RegisterNoiseFlags(cmd.Flags())
 	cmd.Flags().Int32P(FlagMaxDepth, "d", 100, "maximum depth of the tree")
 
 	return cmd

--- a/cmd/expand/root_test.go
+++ b/cmd/expand/root_test.go
@@ -4,6 +4,7 @@ import (
 	"testing"
 
 	"github.com/ory/x/cmdx"
+
 	"github.com/stretchr/testify/assert"
 
 	"github.com/ory/keto/cmd/client"
@@ -15,7 +16,15 @@ func TestExpandCommand(t *testing.T) {
 	ts := client.NewTestServer(t, client.ReadServer, []*namespace.Namespace{nspace}, NewExpandCmd)
 	defer ts.Shutdown(t)
 
-	ts.Cmd.PersistentArgs = append(ts.Cmd.PersistentArgs, "--"+cmdx.FlagFormat, string(cmdx.FormatJSON))
-	stdOut := ts.Cmd.ExecNoErr(t, "access", nspace.Name, "object")
-	assert.Equal(t, "null\n", stdOut)
+	t.Run("case=unknown tuple", func(t *testing.T) {
+		t.Run("format=JSON", func(t *testing.T) {
+			stdOut := ts.Cmd.ExecNoErr(t, "access", nspace.Name, "object", "--"+cmdx.FlagFormat, string(cmdx.FormatJSON))
+			assert.Equal(t, "null\n", stdOut)
+		})
+
+		t.Run("format=default", func(t *testing.T) {
+			stdOut := ts.Cmd.ExecNoErr(t, "access", nspace.Name, "object", "--"+cmdx.FlagFormat, string(cmdx.FormatDefault))
+			assert.Contains(t, stdOut, "empty tree")
+		})
+	})
 }

--- a/contrib/docs-code-samples/expand-api-display-access/99-cleanup/main.go
+++ b/contrib/docs-code-samples/expand-api-display-access/99-cleanup/main.go
@@ -1,3 +1,4 @@
+//go:build docscodesamples
 // +build docscodesamples
 
 package main

--- a/contrib/docs-code-samples/list-api-display-objects/99-cleanup/main.go
+++ b/contrib/docs-code-samples/list-api-display-objects/99-cleanup/main.go
@@ -1,3 +1,4 @@
+//go:build docscodesamples
 // +build docscodesamples
 
 package main

--- a/contrib/docs-code-samples/simple-access-check-guide/99-cleanup/main.go
+++ b/contrib/docs-code-samples/simple-access-check-guide/99-cleanup/main.go
@@ -1,3 +1,4 @@
+//go:build docscodesamples
 // +build docscodesamples
 
 package main

--- a/docs/docs/cli/keto-expand.md
+++ b/docs/docs/cli/keto-expand.md
@@ -28,6 +28,7 @@ keto expand &lt;relation&gt; &lt;namespace&gt; &lt;object&gt; [flags]
   -f, --format string         Set the output format. One of default, json, and json-pretty. (default &#34;default&#34;)
   -h, --help                  help for expand
   -d, --max-depth int32       maximum depth of the tree (default 100)
+  -q, --quiet                 Be quiet with output printing.
       --read-remote string    Remote URL of the read API endpoint. (default &#34;127.0.0.1:4466&#34;)
       --write-remote string   Remote URL of the write API endpoint. (default &#34;127.0.0.1:4467&#34;)
 ```

--- a/go_mod_indirect_pins.go
+++ b/go_mod_indirect_pins.go
@@ -1,3 +1,4 @@
+//go:build go_mod_indirect_pins
 // +build go_mod_indirect_pins
 
 package main

--- a/internal/expand/tree.go
+++ b/internal/expand/tree.go
@@ -162,6 +162,10 @@ func TreeFromProto(t *acl.SubjectTree) (*Tree, error) {
 }
 
 func (t *Tree) String() string {
+	if t == nil {
+		return ""
+	}
+
 	sub := t.Subject.String()
 
 	if t.Type == Leaf {

